plugin/trino-local-file/src/main/java/io/trino/plugin/localfile/LocalFileRecordCursor.java

```diff
@@ -309,7 +310,7 @@ private BufferedReader createNextReader()
             FileInputStream fileInputStream = new FileInputStream(file);
 
             InputStream in = isGZipped(file) ? new GZIPInputStream(fileInputStream) : fileInputStream;
-            return new BufferedReader(new InputStreamReader(in));
+            return new BufferedReader(new InputStreamReader(in, UTF_8));
```

Comment:
i think default charset is appropriate here, since it's a local file with some unspecified charset